### PR TITLE
Update cert-manager to v0.16.1

### LINKED ISSUE #843: Update Cert-manager to v0.16.1

### PR DESCRIPTION
**Pull request by knrt10**
Fixes #843

Signed-off-by: knrt10 <kautilya@kinvolk.io>

### CODE REVIEW

File: assets/charts/components/cert-manager/templates/letsencrypt-clusterissuer-prod.yaml [hunk 1]
```diff
@@ -1,20 +0,0 @@
-apiVersion: cert-manager.io/v1alpha2
```
Comment (invidian) [inline assets/charts/components/cert-manager/templates/letsencrypt-clusterissuer-prod.yaml:1]: This must stay, it's our addition .

File: assets/charts/components/cert-manager/templates/letsencrypt-clusterissuer-staging.yaml [hunk 2]
```diff
@@ -1,20 +0,0 @@
-apiVersion: cert-manager.io/v1alpha2
```
Comment (invidian) [inline assets/charts/components/cert-manager/templates/letsencrypt-clusterissuer-staging.yaml:1]: This must stay, it's our addition .

File: .codespell.skip [hunk 3]
```diff
@@ -2,6 +2,7 @@
 ./.git
```
Comment (invidian) [inline .codespell.skip:2]: > Add cert-manager to .codeskip

It's `.codespell.skip` :smile:

**Comment by surajssd** [comment]
@knrt10 did you encounter any issues during upgrade from the last version?

File: assets/charts/components/cert-manager/Chart.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
```
Comment (surajssd) [inline assets/charts/components/cert-manager/Chart.yaml:1]: A link to release notes in the commit would be helpful.

**Comment by knrt10** [comment]
I only tried applying new one it worked fine, let me try upgrading from previous version

**Comment by knrt10** [comment]
@surajssd upgrade worked fine

cert-manager

**Comment by surajssd** [comment]
@knrt10 the right way to test is if the cert-manager generates certificate after the upgrade.

**Comment by knrt10** [comment]
Yes tested that too

**Comment by knrt10** [comment]
@surajssd 
Screenshot 2020-08-25 at 6 09 19 PM

**Review by surajssd — APPROVED** [review]
ok

**Review by invidian — APPROVED** [review]

**Pull request merged**